src/components/LibraryDetailContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import LibraryDetailPage from "./LibraryDetailPage";
+import { Store } from "redux";
+import { State } from "../reducers/index";
+import { LibraryData } from "../interfaces";
+import { connect } from "react-redux";
+import ActionCreator from "../actions";
+
+export interface LibraryDetailContainerContext {
+  store: Store<State>;
+}
+
+export interface LibraryDetailContainerOwnProps {
+  id: string;
```

Comment:
For consistency why not name this prop `uuid` like the other components?

Reply:
Will do!